remme/rpc_api/transaction.py

```diff
@@ -115,6 +155,24 @@
             message='Signer public key format is not valid'
         )
 
+    current_handler = None
+
+    for handler in TP_HANDLERS:
+        if handler._family_name == tr_head_pb.family_name:
+            current_handler = handler
+            break
+
+    if current_handler:
```

Comment:
There is no `else` branch which allows us not to perform any validation. I think it is better to throw an error here to force devs to write validators.

Reply:
Agree